### DISCUSSION TITLE
Hello .....

### DISCUSSION THREAD

**Discussion by fab1an2** [markqvist/Reticulum#222] [Nomad Network]
I'm starting using nomad network 
How discovered a real people using this program?
Is any free board for sending my adress?

is possible have multiple identification in nomad net? one for family one for outside?

**Comment by markqvist** [comment]
Hi @fab1an2

I can see you're really enthusiastic about all of this, and that's awesome. I'd suggest you read the material posted in the various guides, manuals and help menus of the related programs, they will answer all of the basic stuff. You'll get your answers easier that way. Since we're a small number of people here, there's just not enough time to answer the same basic questions many times.

For the above questions, read the "Guide" section in the Nomad Network program. All the answers are there!

Sure, you can have as many identities as you want.

Once you stay connected to the network a little while, you will start to see stuff in your announce stream too. With that you can start exploring some of the stuff that is out there.

I will not post any specific addresses or such here, since they tend to change. The best way is to simply jump in. And have a *little* patience.

**Reply by fab1an2** [reply]
Are you telling me that anyone can view my communications? Anyone can see that I am sending a message to someone else? This is not anonymous.

**Reply by markqvist** [reply]
> Are you telling me that anyone can view my communications?

No.

**Comment by srem1** [comment]
My adress is 3ecb0a2c062a024ae9aa6201ebfacf94
